### PR TITLE
En 615 login page integration

### PR DESCRIPTION
**Pull request by fedelocci**

### CODE REVIEW

File: __mocks__/api/login.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+import { BODY_OK, BODY_ERROR } from 'test/mocks/login';
+
+// eslint-disable-next-line
+export const login = (username, password) => {
```
Comment (nicpuddu) [inline __mocks__/api/login.js]: if this is the only export lets add a default

File: src/api/login.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+
+import { BODY_OK, BODY_ERROR } from 'test/mocks/login';
+
+// eslint-disable-next-line
+export const login = (username, password) => {
```
Comment (nicpuddu) [inline src/api/login.js]: if this is the online export let add a default

File: src/test/state/form/actions.test.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    message: '',
+  },
+};
+
+const SET_LOGIN_ERROR_MESSAGE_INITIAL_STATE = {
+  type: SET_LOGIN_ERROR_MESSAGE,
+  payload: {
+    message: '',
+  },
+};
```
Comment (nicpuddu) [inline src/test/state/form/actions.test.js]: why two variables with the exact same content?

File: src/test/state/form/actions.test.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+const SET_LOGIN_SUCCESS_MESSAGE_MOCK = {
+  type: SET_LOGIN_ERROR_MESSAGE,
+  payload: {
+    message: '',
+  },
+};
+
+const SET_LOGIN_ERROR_MESSAGE_INITIAL_STATE = {
```
Comment (alessioscalici) [inline src/test/state/form/actions.test.js]: The characters are free, but please try to keep names short :smile:

File: src/state/form/actions.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+// thunks
+
+export const performLogin = (username, password) => (dispatch) => {
+  if (username && password) {
+    dispatch(setLoginErrorMessage(''));
+    login(username, password).then(() => {
+      gotoRoute('dashboard');
+    });
+  } else {
+    dispatch(setLoginErrorMessage('ERROR: USERNAME OR PASSWORD IS INVALID'));
```
Comment (alessioscalici) [inline src/state/form/actions.js]: Please use `formattedText` function and create a message in locales/en and locales/it

**Comment by nicpuddu** [comment]
:ship: :it:

**Pull request merged**